projects/wmf_wikivoyage.yaml

```diff
@@ -0,0 +1,201 @@
+                [Terms of Use](https://wikimediafoundation.org/wiki/Terms_of_Use) and
+                [Privacy Policy](https://wikimediafoundation.org/wiki/Privacy_policy).
+                Unless otherwise specified in the endpoint documentation
+                below, content accessed via this API is licensed under the
+                [CC-BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)
+                and [GFDL](https://www.gnu.org/copyleft/fdl.html) licenses,
+                and you irrevocably agree to release modifications or
+                additions made through this API under these licenses.
+                See https://www.mediawiki.org/wiki/REST_API for background and details.
+
+                ### Endpoint documentation
+
+                Please consult each endpoint's documentation for details on:
+
+                - Licensing information for the specific type of content
+                  and data served via the endpoint.
+                - Stability markers to inform you about development status and
+                  change policy, according to
+                  [our API version policy](https://www.mediawiki.org/wiki/API_versioning).
+                - Endpoint specific usage limits.
+
+            termsOfService: https://wikimediafoundation.org/wiki/Terms_of_Use
+            contact:
+              name: the Wikimedia Services team
+              url: http://mediawiki.org/wiki/REST_API
+            license:
+              name: Apache2
+              url: http://www.apache.org/licenses/LICENSE-2.0
+          # Override the base path for host-based (proxied) requests. In our case,
+          # we proxy https://{domain}/api/rest_v1/ to the API.
+          x-host-basePath: /api/rest_v1
+          x-route-filters:
+            - path: lib/content_location_filter.js
+            - path: ./lib/normalize_title_filter.js
+              options:
+                redirect_cache_control: '{{options.purged_cache_control}}'
+          paths:
+            /page:
+              x-modules:
+                - path: v1/content.yaml
+                  options:
+                    response_cache_control: '{{options.purged_cache_control}}'
+                - path: v1/content_segments.yaml
+                  options:
+                    response_cache_control: '{{options.purged_cache_control}}'
+                    cx_host: '{{options.transform.cx_host}}'
+                - path: v1/mobileapps.yaml
+                  # NOTE: We're setting the `no-store` cache-control header so that we don't
+                  # store MCS results - we don't have pre-generation  for wikivoyage.
+                  options: '{{merge({"response_cache_control": options.purged_cache_control},
+                                    options.mobileapps,
+                                    {"cache-control": "no-store"})}}'
+                - path: v1/graphoid.yaml
+                  options: '{{options.graphoid}}'
+                - path: v1/summary.js
```

Comment:
We probably want the `no-store` here as well, since there is no pregeneration for wikivoyage summaries either.

Reply:
Why? we do pregenerate summaries for almost everything.